src/js/index.js

```diff
@@ -33,6 +34,25 @@ $('#return-to-top').click(function() {
 	}, 500);
 });
 
+//Owl
+$(window).scroll(function() {
+	if ($(this).scrollTop() >= 50) {        
```

Comment:
It doesn't matter as much if you're the only one working on a project, but I'd still recommend using two space characters instead of tabs. That's what Sparkbox uses by default, since you run into fewer cross-platform issues that way (and it's easier on the eyes, in my opinion).